src/dll/v0/Funcs.cpp

```diff
@@ -110,3 +111,21 @@ bool v0::GameStates::Add(RED4ext::PluginHandle aHandle, RED4ext::EGameStateType
     spdlog::warn(L"The request to add a '{}' state for '{}' has failed", Utils::GetStateName(aType), plugin->GetName());
     return false;
 }
+
+bool v0::Scripts::Add(RED4ext::PluginHandle aHandle, const char* aPath)
```

Comment:
I would use `wchar_t` here, what do you think?

Reply:
Sounds good!